doc/ref_model/chapters/appendix-a.md

```diff
@@ -45,25 +53,48 @@ Several specific technology areas have been identified by CNTT as using an ABI i
+
+> **Note to Authors:** Status should be set to "Proposed" when initial content is entered. Once alignment is attained following vetting and discussion, status should be set to "Aligned". Immediately prior to merge, status should be set to "In Force". When amending previously approved language, status should be changed from "In Force" to "In Force (Pending Changes)", followed by "Aligned" and ultimately, "In Force".
+
+**SR-IOV**
+- **Status:** Proposed
+- **Revision:** v0.1 - 8/12/2019 - Shostak
+
+SR-IOV transgresses the CNTT Abstraction Principle, which prohibits hardware-dependent code be required in a workload. Refer to the Abstraction Principle in RM Chapter 1 for the related rationale and details. However, CNTT recognizes that today, SR-IOV provides a capability to increase throughput over network interfaces at an economical cost, that some applications as currently implemented, are highly dependent upon. As such, the CNTT approach to SR-IOV shall be:
+
+> **DISCLAIMER: I (Mark) am typing this proposed language based on input I have received. Therefore, people with strong views may need to discuss this online or live, and come to a consensus. In other words, this is a starting point. Expect there to be discussion and don't shoot the messenger.** (To be removed before merge)
 
-**SR-IOV:**
+1. At the present Time Point (TP), TP x, to enable VNFs currently dependent on SR-IOV to transition to Cloud Native technology, VNFs using SR-IOV will PASS OVP qualification, but generate a WARNING indicating the functionality will be deprecated, and to transition to a Cloud Native implementation
+1. At TP x+y, VNFs using SR-IOV will PASS OVP qualification, but generate a WARNING indicating the functionality is deprecated and they must transition to a Cloud Native implementation
+1. At TP x+y+z, VNFs using SR-IOV will **FAIL** OVP qualification, and generate an **ERROR** indicating the functionality is prohibited
 
-CNTT recognizes that today, SR-IOV provides a critical Capability for increased throughput over network interfaces at an economical cost. As such, the CNTT approach to SR-IOV is to (detailed policy language under development; to be tied in with VNF Evolution).
+NOTE: A Time Point (TP) metric representing chronologic advancement of the CNTT VNF model is being developed in parallel under the VNF Evolution Framework. Hence, TPs referenced herein are placeholders and intended to be relative, until the absolute TPs are defined, at which point the placeholders are to be replaced with their absolute equivalents.
```

Comment:
How would this work when needing to roll out a host hardware or software change across an NFVI?  across all NFVIs?

Reply:
@kedmison Good question; that will be dependent on a plethora of factors that are likely specific to the situation. The purpose of this framework is to identify/capture the technologies, their associated compromises/policy, and bind them to the VNF Evolution "clock", as appropriate. As a given technology advances from state to state (i.e. compromise to compromise), a tailored cutover strategy will be required, but the key here is to identify the "what" (the stages are), the "when" (the stages become active) and the "why" (i.e. rationale). The "how" to effect said stages is more of an operational discussion, than a strategic one. Thanks, -Mark